nni/algorithms/compression/v2/pytorch/pruning/tools/sparsity_allocator.py

```diff
@@ -47,8 +47,8 @@ class BankSparsityAllocator(SparsityAllocator):
     aligned with balance_gran. Each sub block has the same sparsity which equal to the overall sparsity.
     This allocator pruned the weight in the granularity of block. 
```

Comment:
pruned -> prunes

Reply:
maybe better to mention whether this allocator allocates sparsity across layers or only focuses on single weight